modules/custom/az_paragraphs/src/AZVideoEmbedHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\az_paragraphs;
+
+/**
+ * Class AZVideoEmbedHelper. Adds service to help get video metadata.
+ */
+class AZVideoEmbedHelper {
+
+  /**
+   * Get YouTube video id from URL.
+   *
+   * @param string $url
+   *   A YouTube url.
+   *
+   * @return string
+   *   Mixed Youtube video ID or FALSE if not found.
```

Comment:
Nitpick:
```suggestion
   * @return string|bool
   *   String Youtube video ID or boolean FALSE if not found.
```

Reply:
The `@return` line should still say either `mixed` or `string|bool` instead of `string`.